_config.yml

```diff
@@ -13,13 +13,17 @@ github:
 # Configuration for DAP, add your agency ID here:
 # dap_agency: GSA
 
-# To enable AnchorJS functionality on page and post layouts, uncomment this section.
+# Uncomment to enable AnchorJS functionality on docs layout.
 # See https://github.com/bryanbraun/anchorjs for more information.
 # anchor_js_targets: [h1, h2, h3, h4, h5, h6]
 
-# To enable search, uncomment this section.
+# To enable search, uncomment the search section
 # You will need to setup a search account with search.gov
 # https://search.usa.gov/signup
 # Your search_site_handle value is from the search.gov settings page.
 
 # search_site_handle: searchgov-site-handle
+
+sass:
```

Comment:
To avoid needing to include this, can we move `site.scss` to the `_sass` directory rather than the `assets/css` directory?

Reply:
Feel free to move this 👍

Reply:
I'm not sure I follow. The way its configured now is that any CSS added to site.scss will get appended to the compiled main.css file. So if folks want to add custom styles they should be able to just stick them right in site.scss whether its in `assets/css` or `_sass`.